docs/index.rst

```diff
@@ -112,6 +112,7 @@ The OPC UA Advanced tab allows advanced configuration parameters to be set.
     The path does not include the *Objects* folder or the subscribed Node.
   - **Full OPC UA Path meta data name**: The data point name to use when adding the full OPC UA path to every reading. Default is *OPCUAPath*.
   - **Debug Trace File**: Enable the S2OPCUA OPCUA Toolkit trace file for debugging. If enabled, log files will appear in the directory */usr/local/fledge/data/logs*.
+  - **MonitoredItem block size**: The number of MonitoredItem to be added on to the OPCUA server in single call.
```

Comment:
(i) The term _MonitoredItem_ is not defined anywhere so it should be done here. I suggest "MonitoredItem is the OPC/UA term for a single data stream registered for data updates from the OPC/UA server, (ii) since this is now configurable, you should provide some guidance to system managers on how to set it, including a strong recommendation on why they should leave it alone, (iii) the sentence itself. _MonitoredItem_ should be plural, "to be added on to" should be "registered with," "OPCUA" should be "OPC/UA" to match the rest of the page, "single call" should be "single call to the S2OPCUA OPCUA Toolkit."